packages/jest-expo/jest-preset.js

```diff
@@ -0,0 +1,28 @@
+module.exports = {
```

Comment:
prettier makes the diff so big that git thinks it's a new file.

What I've done is add all the `require.resolve`s (and replace `jest-expo/` with `./`)